identity-com/cryptid
# IDCOM-2080 Basic CLI middleware support

CLI support for middleware - using existing cryptid accounts only, not creating new cryptid accounts with middleware.

Change cryptid tx execution commands to use proposeAndExecute instead of directExecute if the current cryptid account has middleware.

New commands:
cryptid accounts show|set - list and choose between cryptid accounts

Refactored:
cryptid config (now uses oclif topics rather than its own command parsing)

How to test:

Easiest way is to create a second cryptid account with middleware:

1. visit vault-demo.surge.sh
2. connect wallet, enter a failsafe in the middleware form, create the middleware account
3. Create the vault (you should now have two, one default, and one with middleware
4. Go to the cli, run `cryptid accounts`

Note, to test on a dummy wallet, create a new cryptid config with 

```
cryptid init -c <PATH_TO_NEW_CRYPTID_CONFIG> -k <PATH_TO_KEYPAIR>
```

and refer to it everywhere e.g.

```
cryptid accounts -c <PATH_TO_NEW_CRYPTID_CONFIG>
```

or set the CRYPTID_CONFIG env var.
